heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/listener/InterceptorListener.java

```diff
@@ -57,7 +57,7 @@
      private InterceptorJDBCRepository interceptorJdbcRepository;
 
      @Autowired
-     private StartServer startServer;
+     private InterceptorContext startServer;
```

Comment:
The name of the variable should have changed aswell.

Reply:
i will